openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -32,20 +36,24 @@ def course_home_url(course):
     )
 
 
-class TestCourseHomePage(SharedModuleStoreTestCase):
+class CourseHomePageTestCase(SharedModuleStoreTestCase):
```

Comment:
Is this a case of needing to rearrange our tests, or do we just not have unit tests for these features?  It seems like a simple single smoke test for each feature would be necessary.  Do we need a story to add this?